### DISCUSSION TITLE
Can't contribute due to access rights

### DISCUSSION THREAD

**Discussion by pergatore** [helix-editor/helix#11469] [Q&A] [answered]
I wanted to contribute a few changes in the wording in the tutor where it could be a bit clearer / less confusing, but when trying to submit a PR I get "Could not read from remote repository, Please make sure you have the correct access rights and the repository exists" Should I instead submit it as a suggestion?

**Comment by the-mikedavis** [comment, marked as answer]
It sounds like you might have a typo in the name of your remote for Helix. Also note that you'll need to fork Helix to send a pull request. GitHub has pretty extensive docs on the whole contributing process here: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests

**Comment by leo-unglaub** [comment]
According to your Github profile you have not forked [helix-editor/helix ](https://github.com/helix-editor/helix) yet (or its non public). You have to fork it, push changes and then create a PR. You cannot to that on the repository directly.

**Comment by pergatore** [comment]
Ah sorry, first time contributing without being part of the project, that was very much it.
